src/DataProtection/Cryptography.Internal/src/UnsafeNativeMethods.cs

```diff
@@ -82,13 +82,11 @@ private static Lazy<SafeLibraryHandle> GetLazyLibraryHandle(string libraryName)
             [In] uint dwFlags);
 
         [DllImport(BCRYPT_LIB, CallingConvention = CallingConvention.Winapi)]
-        [ReliabilityContract(Consistency.WillNotCorruptState, Cer.Success)]
```

Comment:
Chris brought up a good point about still building for netstandard so we still want these probably